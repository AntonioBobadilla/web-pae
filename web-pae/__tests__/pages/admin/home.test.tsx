import Home from '@/pages/admin/home';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import React from 'react';

describe('Admin Home', () => {
  it('renders admin home page unchanged', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
