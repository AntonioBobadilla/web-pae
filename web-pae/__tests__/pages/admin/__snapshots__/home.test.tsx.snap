// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin Home renders admin home page unchanged 1`] = `
<div>
  <div
    class="cards"
  >
    <div
      class="card"
    >
      <div
        class="box"
        href="/admin/tutors"
        role="button"
      >
        <img
          class="image"
          src="/images/tutors.jpg"
        />
        <p
          class="description"
        >
           
          PAE tutors
           
        </p>
      </div>
    </div>
    <div
      class="card"
    >
      <div
        class="box"
        href="/admin/tutorings-requests"
        role="button"
      >
        <img
          class="image"
          src="/images/tutoring-request.jpg"
        />
        <p
          class="description"
        >
           
          Tutoring requests
           
        </p>
      </div>
    </div>
    <div
      class="card"
    >
      <div
        class="box"
        href="/admin/history"
        role="button"
      >
        <img
          class="image"
          src="/images/back.jpeg"
        />
        <p
          class="description"
        >
           
          Tutoring records
           
        </p>
      </div>
    </div>
    <div
      class="card"
    >
      <div
        class="box"
        href="/admin/subjects"
        role="button"
      >
        <img
          class="image"
          src="/images/adminUF.jpg"
        />
        <p
          class="description"
        >
           
          Manage subjects
           
        </p>
      </div>
    </div>
    <div
      class="card"
    >
      <div
        class="box"
        href="/admin/polls"
        role="button"
      >
        <img
          class="image"
          src="/images/survey.jpeg"
        />
        <p
          class="description"
        >
           
          Polls
           
        </p>
      </div>
    </div>
    <div
      class="card"
    >
      <div
        class="box"
        href="/admin/admins"
        role="button"
      >
        <img
          class="image"
          src="/images/administrators.jpg"
        />
        <p
          class="description"
        >
           
          PAE administrators
           
        </p>
      </div>
    </div>
  </div>
</div>
`;
